tests/integration-test/src/tests/utils.rs

```diff
@@ -101,6 +101,23 @@ pub fn start_bpfd() -> Result<ChildGuard> {
     Ok(bpfd_process)
 }
 
+/// Retrieve bpfctl help test
+pub fn get_bpfctl_list_helptext() -> Result<String, anyhow::Error> {
+    //let mut args = Vec::new();
+    //args.push("list");
+    //args.push("--help");
+    let args = vec!["list", "-help"];
+    let output = Command::cargo_bin("bpfctl")?.args(args).ok();
+    match output {
+        Ok(o) => {
+            let stdout = String::from_utf8(o.unwrap().stdout).unwrap();
+            let helptext = stdout.trim();
+            Ok(helptext.to_string())
+        }
+        Err(e) => Err(e.into()),
+    }
+}
+
```

Comment:
Main thing is to remove the comments but you could also simplify like this...

```suggestion
/// Retrieve bpfctl help test
pub fn get_bpfctl_list_helptext() -> Result<String, anyhow::Error> {
    let args = vec!["list", "-help"];
    Command::cargo_bin("bpfctl")?
        .args(args)
        .ok()
        .map(|o| {
            String::from_utf8(o.stdout)
                .unwrap_or(String::new())
                .trim()
                .to_string()
        })
        .map_err(|e| e.into())
}
```

In general though (for next time) it probably makes sense to not even wrap this in a util function since it's generally a one off (i.e let's just make sure bpfctl has a help command 😆 ) 

Then it would turn all of this into like this simple thing 

```rust
#[integration_test]
fn test_bpfctl_helptext() {
    let args = vec!["list", "-help"];

    assert!(!Command::cargo_bin("bpfctl").unwrap()
        .args(args)
        .ok()
        .expect("bpfctl list --help failed")
        .stdout.is_empty());
}
```